pagerduty/event_orchestration_path_util.go

```diff
@@ -223,3 +224,20 @@ func flattenEventOrchestrationPathExtractions(e []*pagerduty.EventOrchestrationP
 	}
 	return res
 }
+
+func convertEventOrchestrationPathWarningsToDiagnostics(warnings []*pagerduty.EventOrchestrationPathWarning, diags diag.Diagnostics) diag.Diagnostics {
```

Comment:
This method is used in the UpdateContext functions of the following resources:
- pagerduty/resource_pagerduty_event_orchestration_path_router.go
- pagerduty/resource_pagerduty_event_orchestration_path_unrouted.go
- pagerduty/resource_pagerduty_event_orchestration_path_service.go

It takes a collection of warnings returned by the Event Orchestration Path Update endpoint and converts them into a collection of [Diagnostics](https://pkg.go.dev/github.com/hashicorp/terraform-plugin-sdk/v2/diag) of type `Warning` that are returned to the Provider user on `terraform apply`.